apps/supernova/README.md

```diff
@@ -40,15 +40,54 @@ Silence information per alert:
 
 We publish a self-hostable docker image [in our registry](https://github.com/cloudoperators/juno/pkgs/container/juno-app-supernova). The README for it can be found [here in the docker folder](https://github.com/cloudoperators/juno/tree/main/apps/supernova/docker)
 
+### Standalone Mode
+
+To create a static, runnable build, execute the following commands:
+
+```bash
+cd apps/heureka
```

Comment:
shouldn't be `supernova` here instead of `heureka`?

Reply:
Good catch! 🤦‍♂️
Fixed!